reader/components/layout/Layout.jsx

```diff
@@ -8,6 +8,7 @@ const Layout = ({
   tag: Tag = 'div',
   ...restProps
 }) => (
+  // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
I am going to use this everywhere. Why do you against using spread for components? It allows write less.

Reply:
The same as https://github.com/oacore/reader/pull/108#discussion_r347131047. In addition, spreading props can be dangerous. I would rather let this rule be and don't change it. Sometimes it's unavoidable and I prefer adding disable comments.